kuba-kostrzewski/simpleToDoApp/src/main/java/pl/kostrzej/simpleToDoApp/components/validator/UserValidator.java

```diff
@@ -0,0 +1,18 @@
+package pl.kostrzej.simpleToDoApp.components.validator;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import pl.kostrzej.simpleToDoApp.components.user.User;
+import pl.kostrzej.simpleToDoApp.components.user.UserRepository;
+
+@Component
+public class UserValidator {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    public void validate(User user){
```

Comment:
Na przyszłość: w przypadku, gdybyś nie pisał frontu (tak jak u Ciebie to wygląda w kontrollerze) to pamiętaj w Validatorze sprawdzać również czy pola nie są puste, mają minimalną lub maksymalną długość. Są to takie podstawowe case.

Reply:
Mimo, że walidacja jest w kontrollerze to dodałem dodatkową walidację w metodzie validate